integration-tests/tests/installApp_test.go

```diff
@@ -89,3 +89,15 @@ func (s *installAppSuite) TestInstallUnexistingAppMustPrintError(c *check.C) {
 			"unexisting failed to install: snappy package not found\n",
 		check.Commentf("Wrong error message"))
 }
+
+func (s *installAppSuite) TestInstallFromStoreMetadata(c *check.C) {
+	cmd := exec.Command("sudo", "snappy", "install", "hello-world/edge")
```

Comment:
Ah, the one above was written before we had support for expected errors. I'll update that one.